transaction.go

```diff
@@ -46,6 +64,21 @@ func (db *DB) Begin() (*DB, error) {
 	return &db2, nil
 }
 
+// Begin starts a new transaction, defaults to write mode
+func (db *DB) Begin() (*DB, error) {
+	return db.txBegin(true)
+}
+
+// BeginWrite starts a new transaction, write mode
+func (db *DB) BeginWrite() (*DB, error) {
+	return db.Begin()
+}
+
```

Comment:
Remove this. It suggests that we can do only writes, no reads, which is not the case.